<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupeGdeImage">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Param√®tres de l'image</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>222</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="labelTaille">
          <property name="text">
           <string>Taille :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="taille">
          <item>
           <property name="text">
            <string>1:1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2:1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3:2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4:3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5:4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16:9</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="labelFrame">
          <property name="text">
           <string>Frame :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="frames">
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="modelColumn">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/frames/0</normalon>
            </iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/frames/1</normalon>
            </iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/frames/2</normalon>
            </iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/frames/3</normalon>
            </iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/frames/4</normalon>
            </iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/5</normaloff>:/frames/5</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/6</normaloff>:/frames/6</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/7</normaloff>:/frames/7</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/8</normaloff>:/frames/8</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/9</normaloff>:/frames/9</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/10</normaloff>:/frames/10</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/11</normaloff>:/frames/11</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/12</normaloff>:/frames/12</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Ressources.qrc">
             <normaloff>:/frames/13</normaloff>:/frames/13</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10"/>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="labelFond">
          <property name="text">
           <string>Fond : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="couleurFond">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnModifierFond">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="labelCadre">
          <property name="text">
           <string>Cadres images :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="couleurCadres">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCouleurCadres">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="labelEpaisseur">
          <property name="text">
           <string>Epaisseur des cadres :   -  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="epaisseur">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRondeur2_2">
          <property name="text">
           <string>  +</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="labelRondeur">
          <property name="text">
           <string>Rondeur des cadres :    -  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="rondeur">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRondeur2">
          <property name="text">
           <string>  +</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupPetiteImage">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Image s√©lectionn√©e</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>278</width>
       <height>180</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="labelPositionnement">
          <property name="text">
           <string>Positionnement :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="btnGH">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16000000</width>
              <height>16000000</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/GH</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnMH">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/MH</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnDH">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/DH</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnGM">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/GM</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnMM">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/MM</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnDM">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/DM</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btnGB">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/GB</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btnMB">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/MB</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btnDB">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/logo/positionnement/DB</normalon>
             </iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="labelRotation">
          <property name="text">
           <string>Rotation :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMoins90">
          <property name="text">
           <string>- 90¬∞</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPlus90">
          <property name="text">
           <string>+90¬∞</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="labelEffet">
          <property name="text">
           <string>Effet :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNoirBlanc">
          <property name="text">
           <string>Noir&amp;Blanc</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNegatif">
          <property name="text">
           <string>Negatif</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSepia">
          <property name="text">
           <string>Sepia</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="monGrandWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>441</width>
      <height>381</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="monGrandLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupDiapo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>651</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Diaporama</string>
    </property>
    <widget class="QPushButton" name="btnAjouter">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>50</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewDiaporama">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>561</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="MonTresGrandWidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>661</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </widget>
   <zorder>MonTresGrandWidget</zorder>
   <zorder>groupPetiteImage</zorder>
   <zorder>groupDiapo</zorder>
   <zorder>monGrandWidget</zorder>
   <zorder>groupeGdeImage</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
